susemanager/bin/pg-migrate-12-to-13.sh

```diff
@@ -124,6 +124,8 @@ chown postgres:postgres /var/lib/pgsql/data/*
 
 echo "`date +"%H:%M:%S"`   Starting spacewalk services..."
 systemctl start postgresql
+echo "Reindexing database. This may take a while, please do not cancel it!"
+database=$(sed -n "s/^\s*db_name\s*=\s*\([^ ]*\)\s*$/\1/p" /etc/rhn/rhn.conf)
+spacewalk-sql --select-mode - <<<"REINDEX DATABASE ${database};"
```

Comment:
```suggestion
spacewalk-sql --select-mode - <<<"REINDEX DATABASE ${database};"
if [ ${?} -ne 0 ]; then
    echo "The reindexing failed. Please review the PostgreSQL logs at /var/lib/pgsql/data/log"
    exit 1
```
If `spacewalk-sql` returns error codes != 0 when there are errors, and = 0 when there are not, that should work just fine.

Reply:
Maybe we should improve the message at the `echo`, to tell the user what to do, in the unlikely event this fails. The script has other examples for when the migration fails. Maybe you just need to copy and paste.

Reply:
And remember to do this to the other script 10-to-13 :-)